### PR TITLE
Design tokens for the Announcement component

### PR DESCRIPTION
**Pull request by mariannuar**
## Summary
Design tokens for the new Announcement component 

## How to review this pull request
- [ ] To review, confirm that Announcement component exists and it has design tokens in [Emulsify UI Kit](https://www.figma.com/design/TuQAuw81mBRsaepz2HP7BX/Emulsify-UI-Kit?node-id=23835-34045&t=NWEa0Aqlzq78AxZA-4)

### CODE REVIEW

File: src/tokens/transformed.tokens.json [hunk 1]
```diff
@@ -21,6 +21,48 @@
       }
     }
   },
+  "background": {
+    "default-bkg": {
```
Comment (amazingrando) [inline src/tokens/transformed.tokens.json:25]: How do you feel about changing this from "default-bkg" to "default" to match the other tokens?

**Review by amazingrando — CHANGES_REQUESTED** [review]
I left one request for a change.

File: src/tokens/transformed.tokens.json (hunk 1 above)
Comment (mariannuar) [reply src/tokens/transformed.tokens.json:25]: Yeeees, that would be better.

**Comment by mariannuar** [comment]
@amazingrando Updated!

File: src/components/tokens/_tokens.scss [hunk 2]
```diff
@@ -157,6 +148,15 @@
   --breadcrumb-text-hover: #192125;
   --block-quote-padding-x: 2rem;
   --block-quote-padding-y: 0.5rem;
+  --background-default: #ffffff;
```
Comment (amazingrando) [inline src/components/tokens/_tokens.scss:151]: @mariannuar I just noticed that the final custom properties for the announcement tokens are missing the namespace.

Here is an example from the Accordion, `--accordion-color-body`.

These new tokens should start with `--announcement-`.

**Review by amazingrando — CHANGES_REQUESTED** [review]
@mariannuar I found another change. Sorry!

File: src/components/tokens/_tokens.scss (hunk 2 above)
Comment (mariannuar) [reply src/components/tokens/_tokens.scss:151]: That's right. I'm sorry, that's my bad 🥲 I'll change it

**Review by amazingrando — APPROVED** [review]
:shipit:

**Pull request merged**